app/src/main/res/layout/fragment_reset_password.xml

```diff
@@ -1,13 +1,103 @@
 <?xml version="1.0" encoding="utf-8"?>
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
This could stay as `LinearLayout`

Reply:
I assumed you change because of the background images. Let's discuss how that can be solved